composer.json

```diff
@@ -173,6 +170,9 @@
             },
             "mglaman/drupal-check": {
                 "Add custom rules to bypass check for buildForm": "https://github.com/mglaman/drupal-check/pull/286.patch"
+            },
+            "stecman/symfony-console-completion": {
+                "Return type fix for stecman/symfony-console-completion": "https://github.com/stecman/symfony-console-completion/pull/94.patch"
```

Comment:
Revert this change, Not required ~~ Merged into starterkit

Reply:
it starterkit new version get released we need this.